wp-content/themes/pub/wporg-learn-2020/sidebar-lesson-plan.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * The sidebar containing the lesson plans widget area.
+ *
+ * @link https://developer.wordpress.org/themes/basics/template-files/#template-partials
+ *
+ * @package WordPressdotorg\Theme
+ */
+
+namespace WordPressdotorg\Theme;
+
+?>
+<aside class="lp-sidebar">
+	<?php
+	if ( is_active_sidebar( 'wporg-learn-lesson-plans' ) ) :
+		dynamic_sidebar( 'wporg-learn-lesson-plans' );
+	endif;
+	?>
```

Comment:
🤔 I wonder if the dynamically added widgets should go _before_ or _after_ the hard-coded stuff? @hlashbrooke any thoughts there?

Reply:
Dynamic widgets after the hard-coded items would be better.

Actually, first prize would be it's all dynamic and we have a block we can use for the stuff that is currently hard-coded. I realise that is a bunch more work though :D

Reply:
👍 

Yeah, let's save the block development for later. Feel free to open an issue about it, though!

Reply:
I have done this.